src/Symfony/Component/Form/DependencyInjection/FormPass.php

```diff
@@ -80,16 +91,26 @@ public function process(ContainerBuilder $container)
             $typeExtensions[$extendedType][] = $serviceId;
```

Comment:
shouldn't we just do `$typeExtensions[$extendedType][] = new Reference($serviceId);` here and remove the `array_map` below?